src/cli/aspencli.py

```diff
@@ -317,6 +316,20 @@ def download_samples(ctx, sample_ids):
     print(resp.text)
 
 
+@samples.command(name="delete")
```

Comment:
I want to clarify: for both the sample deletion and the phylo run deletion, right now the user identity kicking off the CLI command has to be from the same group as the sample/run that is targeted for deletion for it to actually go through, right? At this moment, admins don't get super-user privilege for being able to delete from any group?

Reply:
Right now there are no super-user privs. I'll check with Product as to what we want to do about that.